config/path_config.py

```diff
@@ -0,0 +1,2 @@
+"""Contains dataset's directory path"""
+DATA_DIR = '/home/kuba/Development/Datasets/PetFinder'
```

Comment:
We agreed not to push this to master branch.
fyi: https://stackoverflow.com/questions/4308610/how-to-ignore-certain-files-in-git/44132149

Reply:
fck sry